subsys/partition_manager/Kconfig

```diff
@@ -119,6 +119,17 @@ config PM_SINGLE_IMAGE
 
 DT_CHOSEN_EXT_FLASH:= nordic,pm-ext-flash
 
+config PM_EXTERNAL_FLASH_HAS_DRIVER
+	bool
+	default y if NORDIC_QSPI_NOR || SPI_NOR
+	default y if PM_OVERRIDE_EXTERNAL_DRIVER_CHECK
```

Comment:
I'm not able to find this option. Where is it declared?

Reply:
Same file, line 156 or around.